controller/release.go

```diff
@@ -63,6 +63,15 @@ func (r *ReleaseRepo) Add(data interface{}) error {
 		release.ArtifactIDs = []string{release.LegacyArtifactID}
 	}
 
+	for key := range release.Env {
+		if key == "" {
```

Comment:
Since `release.Env` is a map, you can query for the key directly:

```go
if _, ok := release.Env[""]; ok {
	// empty key exists
	return ...
}
```